src/components/UnifiedWalletButton/index.tsx

```diff
@@ -47,27 +46,17 @@ export const UnifiedWalletButton: React.FC<{
   );
 
   const handleClick = useCallback(async () => {
-    try {
-      if (wallet?.adapter?.name === SolanaMobileWalletAdapterWalletName) {
-        await connect();
-
```

Comment:
We don't try to connect via MWA? Isn't this important for Saga?

Reply:
Connect Wallet buttons conventionally completely skip the modal UI and connect to MWA directly. It's not gonna work for us:

1. Connecting a second time on android does **nothing** and is undetectable
2. Backpack doesn't support MWA
3. Lets be real, 500 people use Saga today

The Mobile adapter will be found in the list